lk2nd/device/dts/msm8994/msm8994-huawei-angler.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8994 0>;
+	qcom,board-id = <8026 0>;
+	qcom,pmic-id = <0x10009 0x1000a 0x00 0x00>;
+};
+
+&lk2nd {
+	model = "Huawei Nexus 6P";
```

Comment:
Could you please also add it to the devices.md in the same commit, and reword the commit subject in line with usual prefixes? (i.e. add `dts: msm8994: `)